davalebbba9.js

```diff
@@ -0,0 +1,90 @@
+// სავარჯიშო 1 
+
+let m = 160;
+n = 17;
+
+let x1 = didiricxvi(m, n);
+//x არის მეტი რომელიცაა
+function didiricxvi(m, n) {
+   if (m > n) {
+      x = m;
+   }
+   else if (n>m ) {
+      x =n;
+   } else {
+      x = 0
+   }
+}
+console.log(x);
+
+
+
+// სავარჯიშო 2
+addnumbers (12, 2);
+function addnumbers(a,b) {
+console.log("რიცხვების ჯამია", a+b);
+}
+
+
+// სავარჯიშო 3
+function fullName(name, surname){
+console.log("მე ვარ", name, surname);
+}
+fullName("Nino ", "Khositashvili");
+
+
+// სავარჯიშო 4
+
+let me ={
+   myName: "Nino", 
+   mySurname: "Khositashvili",
+   fullName: function(){
+   console.log(this.myName, this.mySurname)
+}
+}
+me.fullName();
+
+
+
+
+// სავარჯიშო 5
```

Comment:
function multiply (n) {
  let namravli= 1;
    for (let i=1; i<=n; i++){
       namravli *= i;
    }
    return namravli
 }

 console.log(multiply (3));